wright-filter-wheels/wright_filter_wheels/_yaqd_wright_filter_wheels_discrete.py

```diff
@@ -0,0 +1,101 @@
+__all__ = ["YaqdWrightFilterWheelsDiscrete"]
+
+import asyncio
+from typing import Dict, Any, List
+from yaqd_core import DiscreteHardware, aserial
+
+from .__version__ import __branch__
+
+class YaqdWrightFilterWheelsDiscrete(DiscreteHardware):
+    _kind = "wright-filter-wheels-discrete"
+    _version = "0.1.0" + f"+{__branch__}" if __branch__ else ""
+    traits: List[str] = ["uses-uart","uses-serial","is-homeable"]
+    defaults: Dict[str, Any] = {"baud_rate": 57600}
+    
+    def __init__(self, name, config, config_filepath):
+        super().__init__(name, config, config_filepath)
+        self._motornum=config["motor"]
+        self._serial_port = aserial.ASerial(config["serial_port"], config["baud_rate"])
+        self.microstep=1
+        self._steps_per_rotation=400
+        self._current_identifier=""
+        self._offset=0  # try incorporating any adjustments to TOML before adjusting this
+        self._identifiers=config["identifiers"]
+        self.home()
+
+
+    def _load_state(self, state):
+        """Load an initial state from a dictionary (typically read from the state.toml file).
+
+        Must be tolerant of missing fields, including entirely empty initial states.
+
+        Parameters
+        ----------
+        state: dict
+            The saved state to load.
+        """
+        super()._load_state(state)
+        # This is an example to show the symetry between load and get
+        # If no persistent state is needed, these unctions can be deleted
+        self.value = state.get("value", 0)
+
+    def get_state(self):
+        state = super().get_state()
+        state["value"] = self.value
+        return state
```

Comment:
Setting and loading an unused "dummy" variable.

If you do not need any persistent state, you can delete both get_state and _load_state

Reply:
Reserving load_state and get_state code as I would like to have the state TOML save the current position and identifier (same with continuous), and haven't gone over state file loading yet.

Reply:
Having these methods does not have any effect on whether the state is saved... having them only adds an otherwise unused value to the state file which is saved

Reply:
get_state now removed, could not see it in use for this particular daemon